src/ids-host.test.ts

```diff
@@ -25,6 +25,58 @@ async function mkPromise<T>(lookup: () => T): Promise<T> {
   return lookup();
 }
 
+describe("isUrlSubjectToDynamicUrls", () => {
+  type TestCase = {
+    inputUrl: string;
+    inScope: boolean;
+  };
+
+  const testCases: TestCase[] = [
+    {
+      inputUrl: "https://example.com/bar",
+      inScope: false,
+    },
+    {
+      inputUrl: "https://install.determinate.systems/aoeu",
+      inScope: true,
+    },
+    {
+      inputUrl: "https://install.determinate.systems/",
```

Comment:
I'd also want to test what happens if it doesn't have a trailing `/` (but maybe that's just testing the `URL` type rather than this new functionality)

Reply:
Done!